package exceptions;

public class RunTimeException {

    public static void main(String[] args){
        int myArray[] = {1, 2, 3};
        System.out.println(myArray[3]);
    }
}
